src/helper/archivist.rs

```diff
@@ -125,7 +125,8 @@ pub fn find_marker(location: &Path, marker_name: &str) -> Result<PathBuf> {
     match location.parent() {
         Some(parent) => find_marker(parent, marker_name),
         None => Err(Error::FindMarkerError(
-            "traversed up to fs root, no marker found; maybe go somewhere else using cd?".to_string(),
+            "traversed up to fs root, no marker found; maybe go somewhere else using cd?"
```

Comment:
Error messages should only state the problem, not what the program has tried and failed to do, and should only offer solutions if they are non-trivial.  Git does this good imho: "fatal: not a git repository (or any of the parent directories)". (when you execute a git comment somewhere where there is no git repository)

I think we can assume that "Go somewhere that is actually an experiment" is trivial.